pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>spring-boot-starter-parent</artifactId>
         <groupId>org.springframework.boot</groupId>
-        <version>2.2.0.RELEASE</version>
+        <version>2.3.1.RELEASE</version>
```

Comment:
why the huge jump in spring versions?

Reply:
This was added as a test in order to get the trials working. I don't think it's actually relevant to the problem so will try and remove it (and the other pom changes) and run the trials locally.

Reply:
Removing this does not affect the performance trials. I have therefore removed it (and the palisade and resource pom updates)